Bukkit/src/main/java/com/plotsquared/bukkit/listener/EntitySpawnListener.java

```diff
@@ -140,6 +140,12 @@ public void creatureSpawnEvent(EntitySpawnEvent event) {
                     event.setCancelled(true);
                 }
             }
+            if (type == EntityType.DROPPED_ITEM) {
+                if (Settings.Enabled_Components.KILL_ROAD_ITEMS) {
+                    event.setCancelled(true);
+                }
+                return;
+            }
```

Comment:
This is basically just duplicated with the code a few lines above. I think we can just put it above the `!area.isMobSpawning()` check.

Also note that there is very similar code in `PaperListener` which should probably updated too.

Reply:
Nope, the CreatureSpawnEvent is not called, therefor the PaperListener does not need any update
Tested my changes on paper, and it did work

Reply:
I think it would still make sense to keep them consistent, otherwise that might be confusing in future (as the code isn't documented at all anyways).